Add Community Contributors 2022: Elite Nominators news post (#8816)
ppy/osu-wiki: news/2023/2023-02-17-community-contributors-elite-nominators-2022.md

Through commitment to his craft and a clear vision for the future of mapping, Mirash has helped populate the Ranked section with diversity and quality, all while being spotless compared to his peers, absolutely warranting his spot in the Elite Nominator line-up for this year.

[![](/wiki/shared/news/2023-02-17-community-contributors-elite-nominators-2022/moonpoint.jpg)](https://osu.ppy.sh/users/9558549)

Continuing his excellent performance from 2021, [moonpoint](https://osu.ppy.sh/users/9558549) (formerly known as Apo11o) has proven himself time and time again as a prudent nominator. He's constantly pushed the Ranked section in innovative ways, promoting many map styles and song genres that would likely be ignored otherwise. This was especially notable in 2022, with Apo11o scoring above most other Beatmap Nominators in terms of variety and total beatmap set length, even going out of his way to spot issues in qualified maps and reset their nominations on many occasions.

> moonpoint's spotless tenure as a Beatmap Nominator cannot (and will not) go unnoticed, leaving his spot among the Elite an inevitability.

Reviewer: first paragraph has `Apo11o`, but second starts with `moonpoint`, seems like it should be consistent

He has also run some of the most successful mapping contests the mode has ever seen in the form of [TFAC](/wiki/Contests/o%21tFAC) and [M2P](/wiki/Contests/M2P), and created a [Discord bot](https://osu.ppy.sh/community/forums/topics/1604925) for mapping and modding related statistics which has seen prevalence in over 380 Discord servers! 

Not to mention that this was all done while consistently maintaining his leadership in Project Loved, Beatmap Spotlights, and the Community Mentorship Program. We hope to keep him kidnapped at the NAT headquarters for as long as possible, as who knows what could go wrong if we set him loose upon the world without restraint. 

[![](/wiki/shared/news/2023-02-17-community-contributors-elite-nominators-2022/meiqth.jpg)](https://osu.ppy.sh/users/12565402)

Joining the Beatmap Nominators in June 2021, [meiqth](https://osu.ppy.sh/users/12565402) has been consistently performing at a top level with zero signs of slowing down, and to the surprise of no one, they took 2022 by storm with a consistently excellent performance in nominating a very diverse catalogue of over 200 beatmaps, almost entirely without issues. 

> Not only satisfied with merely that, they also went out of their way to be one of the very few people who actively tackled the daunting task of Quality Assurance in order to make sure no sneaky unrankables or quality issues accidentally make their way into the Ranked section, nipping problems in the bud long before they ever get to inconvenience players.

Reviewer 2: ```suggestion
Not only satisfied with merely that, they also went out of their way to be one of the *very* few people who actively tackled the daunting task of Quality Assurance in order to make sure no sneaky unrankables or quality issues accidentally make their way into the Ranked section, nipping problems in the bud long before they ever get to inconvenience players.
```

would like to emphasize "very" cuz barely anyone else did qah with a decent pace in general

Maxus took massive strides to make mania a lot more active as a nominator in 2022, and we hope to see the same great strides from him in 2023 as well!

[![](/wiki/shared/news/2023-02-17-community-contributors-elite-nominators-2022/gzdongsheng.jpg)](https://osu.ppy.sh/users/8660315)

Being another Mania powerhouse coming into the scene in early 2022, gzdongsheng has been a prime example of quality for other nominators by nominating various high level maps, maps with huge spreads and lengthy drain times, and creations from different mappers across the community. Reaching over 103 nominations, he has proven himself to be an unstoppable force, opening the gates for much more diverse content to enter ranked, all backed up by an impressively consistent modding and nomination quality across the board to boot.

> He is one of the few nominators brave enough to push novel high level 4K maps prone to be controversial and that would otherwise have a difficult time getting into the ranked section. Despite the skill ceiling, he manages to work on them skillfully, showcasing his experience and expertise as a nominator. With the amount of dedication and effort that he put in addition to promoting a variety of keymodes, gzdongsheng has proven himself to be one of the top nominators that are more than deserving to be awarded the coveted title of Elite Nominator!

Reviewer 3: ```suggestion
He is one of the few nominators brave enough to push novel and high-level 4K maps prone to controversy that would otherwise have a difficult time getting into the Ranked section. Despite the skill ceiling, his ability to work on them skilfully showcases his experience and expertise as a nominator. With his amount of dedication and effort, in addition to promoting a variety of keymodes, gzdongsheng has proven himself to be one of the top nominators that are more than deserving to be awarded the coveted title of Elite Nominator!
```

[![](/wiki/shared/news/2023-02-17-community-contributors-elite-nominators-2022/Akasha-.jpg)](https://osu.ppy.sh/users/2596306)

> Breakthrough, obligation, willpower, meticulousness, and affinity might be the 5 best keywords to describe his doings during 2022. Being an experienced Beatmap Nominator with 3 years of long-standing experience, his performance this year is particularly notable. One particularly notable breakthrough of his nominations is the nomination of the iconic beatmap [Last Wish](https://osu.ppy.sh/beatmapsets/1866317#mania/3848743), which went on to achieve the highest star rating of any osu!mania ranked map to date.

Reviewer 3: ```suggestion
Breakthrough, obligation, willpower, meticulousness and affinity might be the 5 best keywords to describe his doings during 2022. As a Beatmap Nominator with a long-standing experience accumulated over 3 years, his performance this year is particularly notable. One particularly notable breakthrough nomination of his was for the iconic [Kry.exe feat. Ice - Last Wish](https://osu.ppy.sh/beatmapsets/1866317), which went on to achieve the highest star rating of any osu!mania Ranked map to date.
```

> Looking back, he has nominated over 100 maps across all keymodes and various styles, covering an extraordinarily wide range of creations. It also includes many challenging large full spread sets, with some even reaching over 10 difficulties at 4 minutes for each one. Never once shirking a friendly and thoughtful attitude, his accomplishments speak for themselves, and make him perfectly suited for the title of Elite Nominator. 

Reviewer 3: ```suggestion
Looking back, he has nominated over 100 maps across all keymodes and various styles, covering an extraordinarily wide range of creations. It also includes many large and challenging full-spread sets, with some even reaching over 10 difficulties at 4 minutes each. Never once shirking a friendly and thoughtful attitude, his accomplishments speak for themselves, and make him perfectly suited for the title of Elite Nominator.
```

[![](/wiki/shared/news/2023-02-17-community-contributors-elite-nominators-2022/secXcscX.jpg)](https://osu.ppy.sh/users/13543418)

> Despite only contributing for a short few months during 2022, secXcscX's main strength shines in full  through the fact that he manages to consistently deliver high-value nominations, which nets him to be one of the highest performing nominators in the entire mania community. One can see his work involving huge spreads and lengthy maps, and it's worth mentioning their diversity in helping a wide pool of unique mappers while we're at it.

Reviewer 3: ```suggestion
Despite only contributing for a few short months during 2022, secXcscX's main strength shines in full through the fact that he manages to consistently deliver high-value nominations, which nets him to be one of the highest-performing nominators in the entire osu!mania community. One can see his work involving huge spreads and lengthy maps, and it's worth mentioning their diversity in helping a wide pool of unique mappers while we're at it.
```

> Scoring a grand total of 91 nominations, secXcscX is the primary initial early force of the content boost in the community that really opened up plenty of opportunities for new mappers to get their first ranked map during 2022's great revival, and also for lengthy maps that normally would have gone unnoticed due to the large effort involved in checking them properly. 

Reviewer 3: ```suggestion
Scoring a grand total of 91 nominations, secXcscX is the primary, initial, and early force of the content boost in the community that really opened up plenty of opportunities for new mappers to get their first Ranked map during 2022's great revival. The same holds true for lengthy maps that normally would have gone unnoticed due to the large effort involved in checking them properly. 
```

> His nominations are consistently high quality at a level that we think makes him an ideal role model for others in the community, and it is in all these traits together that we figure secXcscX is well deserving of the Elite Nominator title.

Reviewer 3: ```suggestion
His nominations are consistently at a high quality to a level that we think makes him an ideal role model for others in the community, and it is in all these traits together that we figure secXcscX is well-deserving of the Elite Nominator title.
```

---

> A big thanks goes out to the NAT as a whole, who came together this year to make this happen, from organizing the recipients to writing out their accolades and organizing much of the process.

Reviewer 3: whatever "organising the recipients" would mean

```suggestion
A big thanks goes out to the NAT as a whole, having come together this year to make this happen, for everything from helping decide the recipients, to writing out their accolades and organising much of the process.
```

We're very thankful to be the hosts for such an engaged and enthusiastic community. Part of what makes osu! awesome is our ancient tradition of huge community contributions dating back well over a decade, all held aloft by a mutual passion for free and fair rhythm gaming. We wouldn't be what we are without you!

As always, we're enormously thankful for all those who contribute, and we look forward to one day sharing your accolades with the greater community.

> —the osu!team

Reviewer 3: ```suggestion
—the osu! team
```
